src/components/sidebar-menu-item/sidebar-menu-item.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./sidebar-menu-item.scss";
+
+export const SidebarMenuItem = ({ active, isTitle, children, onClick }) => {
+  const onClickCb = () => {
+    if (onClick) {
+      onClick();
+    }
+  };
+
+  return (
+    <li
+      className={`__sidebar-menu-item ${active && "active"} ${isTitle &&
+        "isTitle"}`}
+      onClick={onClickCb}
```

Comment:
Couldn't we just pass `onClick` here ?

Reply:
We do it this way so we can check if the `onClick` prop is defined. We could also have the if statement inline, but it's cleaner this way IMO.